content/blog/2021/10/new-architecture-to-ease-kubewarden-administrators-lives.md

```diff
@@ -0,0 +1,259 @@
+---
+title: A new architecture to ease Kubewarden administrators' lives
+authors:
+    - Víctor Cuadrado Juan
+date: 2021-10-01
+---
+
+We are pleased to announce a new architecture for the Kubewarden stack, in line
+with its journey to maturity:
+
+The introduction of a **PolicyServer** Custom Resource Definition which allows
+users to describe a policy-server Deployment, together with binding
+**ClusterAdmissionPolicies** to a specific **PolicyServer** instance.
+
+These 2 changes are accompanied by a multitude of improvements to make Kubewarden
+more comfortable for Kubernetes Administrators, such as validation for
+Kuberwarden Custom Resources, improvements in Helm Charts, Status and
+Conditions for **ClusterAdmissionPolicies**.
+
+For a comprehensive list, see the [release notes](TODO) of our latest Helm
+charts.
+
+## How does it look like?
+
+In previous versions, the Kubewarden Controller instantiated a single Deployment
+of policy-server. That policy-server was configured via a ConfigMap, which
+contained the Deployment options (image, replicas, ...), and a list of policies to
+load compiled from the instantiated ClusterAdmissionPolicies, with information
```

Comment:
```suggestion
contained the Deployment options (image, replicas, ...), and a list of policies to be loaded, with information
```
`list of policies to load compiled from the instantiated ClusterAdmissionPolicies` is not very clear to me

Reply:
Fixed in https://github.com/kubewarden/kubewarden.io/pull/33/commits/fe3faa37d1105e3cb847c9d79b24911697ec20be.